packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.tsx

```diff
@@ -55,6 +55,27 @@ function reducer(state: PatientSearch, action: Action): PatientSearch {
   }
 }
 
+function useOnClickOutside<T extends HTMLElement = HTMLElement>(handler: (event: Event) => void, active = true) {
+  const ref = useRef<T>(null);
+
+  useEffect(() => {
+    if (active) {
+      const listener = (event: Event) => {
+        const el = ref?.current;
+
+        if (el?.contains(event.target as Node) === false) {
```

Comment:
hello @brandones @ZacButko 
The tweak i made here was to get execute the handler each time a click is made on components beyond the referenced.

I have created a thread https://openmrs.slack.com/archives/CHP5QAE5R/p1648509416725489 requesting to add an optional boolean to revert/flip the logic of [useOnClickOutside](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-react-utils/src/useOnClickOutside.tsx), and i kindly request for your suggestions.